### PR TITLE
Promote ConsolePlugin API version to v1

### PR DESCRIPTION
**Pull request by ciiay**
**What type of PR is this?**

/kind bug

**What does this PR do / why we need it**:
ConsolePlugin API has been promoted to v1, we need to update the console plugin CR to pick up the changes.
It's related to [GITOPS-2646](https://issues.redhat.com/browse/GITOPS-2646) to enable i18n in upgraded version.

**Which issue(s) this PR fixes**:

Fixes GITOPS-2646

**Test acceptance criteria**:

* [x] Unit Test
* [x] E2E Test

**How to test changes / Special notes to the reviewer**:
Test ConsolePlugin works.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -113,7 +112,6 @@ func main() {
 	registerComponentOrExit(mgr, operatorsv1alpha1.AddToScheme)
 	registerComponentOrExit(mgr, argoapi.AddToScheme)
 	registerComponentOrExit(mgr, configv1.AddToScheme)
-	registerComponentOrExit(mgr, consolepluginv1.AddToScheme)
```
Comment (jaideepr97) [inline main.go:116]: @ciiay did you mean to update or remove this line? without it the console plugin cannot be deployed at all 
please add it back and update the reference

**Comment by varshab1210** [comment]
Closing this, please reopen if required

**Pull request closed without merging**